### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by antoniukivan**

### CODE REVIEW

**Review by yurwing — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: what about ArrayIndexOutOfBoundException? let’s catch it here as well

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                System.out.println("Будь ласка, введіть корректні дані");
+            }
+            betDao.add(bet);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let’s save  the bet if it is not null

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]:
```suggestion
            System.out.println(bet);
```

File: src/main/java/core/basesyntax/dao/BetDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Bet;
+import java.util.List;
+
+public class BetDaoImpl implements BetDao {
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/dao/BetDaoImpl.java]: redundant empty line

File: src/main/java/core/basesyntax/dao/UserDaoImpl.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+
+public class UserDaoImpl implements UserDao {
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/dao/UserDaoImpl.java]: same

File: src/main/java/core/basesyntax/model/User.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class User {
+    private String login;
+    private int id;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]: let's not have an id for now
in future all our entities will have an id, not only user
```suggestion
    private int age;
```

File: src/main/java/core/basesyntax/model/User.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class User {
+    private String login;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]:
```suggestion
    private String name;
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/model/User.java [hunk 8]
```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class User {
+    private String name;
+    private int age;
+
+    public User(String login, int id) {
+        this.name = login;
+        this.age = id;
+    }
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]: fix variable names

File: src/main/java/core/basesyntax/model/User.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "login='" + name + '\''
+                + ", id=" + age
+                + '}';
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]: here as well

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/model/User.java (hunk 9 above)
Comment (antoniukivan) [reply src/main/java/core/basesyntax/model/User.java]: my bad)

File: src/main/java/core/basesyntax/model/User.java [hunk 10]
```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class User {
+    private String name;
+    private int age;
+
+    public User(String name, int id) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]: id -> age

File: src/main/java/core/basesyntax/model/User.java [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "name = '" + name + '\''
+                + ", id = " + age
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]:
```suggestion
                + ", age = " + age
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**